lib/FFI/Platypus/Lang/Win32.pm

```diff
@@ -193,15 +233,32 @@ sub native_type_map
       }
     }
 
-    # stuff we are not yet dealing with
-    # LPCTSTR is unicode string, not currently supported
-    # LPWSTR 16 bit unicode string
-    # TBYTE TCHAR UNICODE_STRING WCHAR
+    # We don't deal with WCHAR. It's simply a uint16_t, but it's
+    # semantically a Unicode Code Point. This is no biggie since one
+    # should never encounter a WCHAR outside of a LPWSTR or LPCWSTR.
```

Comment:
Lets make WCHAR an alias for `unsigned short`.  It may not be used often, but it doesn't hurt to expose it here.

Reply:
This matches how `char` and `wchar_t` work.